### PR TITLE
satellite/vouchers: deprecate voucher endpoint, return 'please upgrade' error

### PR DESCRIPTION
**Pull request by cam-a**
What: Since we are removing kademlia, we no longer need vouchers. We have already removed the voucher request service on the storage node. This PR removes the logic inside the satellite vouchers endpoint and instead returns a custom error asking the storage node to upgrade to the latest version

Why: We have already removed the storage node voucher request service, but not all storage node operators will have updated to this version. If we remove the satellite vouchers endpoint entirely, those who have not upgraded will see errors and ask the community for help. Returning a custom error asking the storage node to upgrade should remedy the situation

Please describe the tests:
 - Test 1: A test to verify the correct error is returned to the client
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: satellite/vouchers/endpoint.go [hunk 1]
```diff
@@ -5,81 +5,16 @@ package vouchers
...
-	return &Endpoint{
-		log:        log,
-		satellite:  satellite,
-		overlay:    overlay,
-		expiration: expiration,
-	}
-}
+type Endpoint struct{}
 
 // Request receives a voucher request and returns a voucher and an error
```
Comment (JessicaGreben) [inline satellite/vouchers/endpoint.go]: update comment

File: satellite/vouchers/endpoint.go [hunk 2]
```diff
@@ -5,81 +5,16 @@ package vouchers
...
-	"storj.io/storj/pkg/signing"
-	"storj.io/storj/satellite/overlay"
 )
 
-// Config contains voucher endpoint configuration parameters
-type Config struct {
-	Expiration time.Duration `help:"length of time before a voucher expires" default:"720h0m0s"`
-}
-
 // Endpoint for issuing signed vouchers
```
Comment (JessicaGreben) [inline satellite/vouchers/endpoint.go]: could we update this comment and say this is deprecated

File: satellite/vouchers/endpoint.go [hunk 3]
```diff
@@ -5,81 +5,16 @@ package vouchers
...
-	voucher, err := signing.SignVoucher(ctx, endpoint.satellite, unsigned)
-	if err != nil {
-		return nil, Error.Wrap(err)
-	}
-
-	return &pb.VoucherResponse{
-		Voucher: voucher,
-		Status:  pb.VoucherResponse_ACCEPTED,
-	}, nil
+	return nil, errs.New("Vouchers endpoint is deprecated. Please upgrade your storage node to the latest version.")
```
Comment (jenlij) [inline satellite/vouchers/endpoint.go]: will this cause any crashing downstream?

Comment (cam-a) [reply satellite/vouchers/endpoint.go]: It shouldn't. The old storage node voucher service just logs errors and continues running

Comment (jenlij) [reply satellite/vouchers/endpoint.go]: ok great

**Pull request merged**